runtime/compiler/codegen/J9TreeEvaluator.cpp

```diff
@@ -35,6 +35,131 @@
 #include "runtime/J9ValueProfiler.hpp"
 #include "env/PersistentCHTable.hpp"
 
+/*
+ * Returns a J9Method object corresponding to the given ByteCodeInfo.
+ */
+TR_OpaqueMethodBlock *
+J9::TreeEvaluator::getMethodFromBCInfo(TR_ByteCodeInfo &bcInfo, TR::Compilation *comp)
```

Comment:
cab2856

Reply:
Why not just take care of these now? It's a simple enough task and we are already in this area of the code.

Reply:
There's a lot of redundancy here actually and it would make more sense to do this work in a separate Pull Request. I've opened an issue to track this here:
https://github.com/eclipse/openj9/issues/5257